activities/codebase-stewardship/success-for-a-codebase.md

```diff
@@ -4,6 +4,8 @@ type: resource
 
 # Success for a codebase
 
-This is how we define the sucess of codebase stewardship on a codebase:
+We define a succesfully stewarded code base as trustworthy, usable, maintained and sustainable code for your public organization.
```

Comment:
I wonder if this is at odds with the definition below where we say we think a successful codebase is one that is being collectively maintained.

Reply:
Good point - I airlifted this sentence from another location. I guess this is the multiple perspectives issue. Success from the community's perspective, our perspective, or our member's perspective?

Reply:
patched bug